CHANGELOG.md

```diff
@@ -1,5 +1,92 @@
 # Changelog
 
+## [v0.62.0-rc.0.0]
+
+### Breaking
+
+- Removing <TextInput>'s `onTextInput` event ([3f7e0a2](https://github.com/facebook/react-native/commit/3f7e0a2) by [@shergin](https://github.com/shergin))
+- Restore behavior for `underlayColor={null}` in `TouchableHighlight`. ([37d8440](https://github.com/facebook/react-native/commit/37d8440) by [@yungsters](https://github.com/yungsters))
```

Comment:
While I look through the change logs, I was a bit confused with this. New `TouchableHighlight` was introduced in `0.62-rc.0` which had changed default behavior of `underlayColor`. The default behavior has been restored also in `0.62-rc.0`. So, I am not sure if it's a part of breaking changes because anybody who would upgrade rn from published version could not recognize the difference. (I've tested this with 0.61-stable and 0.62-rc.1, and the result between the two is same).

Reply:
Yeah, this may have been a revert of behavior in which case both log lines should typically be removed

Reply:
@TheSavior sorry, I'm confused. Should I remove both "breaking" lines? Just the one about underlayColor?

Reply:
I think it'd be better to keep the first one, which will be helpful for some people who have been using the event even not documented because it might be a breaking change for those people.

Reply:
I think we can remove this line, as it is essentially a revert that was a part of another commit (flagged as internal) that accidentally had a breaking change